### PR TITLE
[INLONG-5657][Release] Change the tag of Docker images to 1.3.0

### LINKED ISSUE #5657: [Release] Change the tag of Docker images to 1.3.0
### Description

[Release] Change the tag of Docker images to 1.3.0

### Use case

[Release] Change the tag of Docker images to 1.3.0

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EMsnap**
- Fixes #5657 

### Motivation

[INLONG-5657][Release] Change the tag of Docker images to 1.3.0

### Modifications

[INLONG-5657][Release] Change the tag of Docker images to 1.3.0

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docker/publish-by-arch.sh [hunk 1]
```diff
@@ -51,15 +51,15 @@ buildImage() {
 initTagImageForx86() {
   SRC_POSTFIX=""
   DES_POSTFIX="-x86"
```
Comment (dockerzhang) [inline docker/publish-by-arch.sh:53]: this file is not needed to change, the version is defined by `MVN_VERSION`.

File: inlong-agent/agent-common/pom.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.inlong</groupId>
         <artifactId>inlong-agent</artifactId>
-        <version>1.3.0-SNAPSHOT</version>
```
Comment (dockerzhang) [inline inlong-agent/agent-common/pom.xml:26]: the version changing in pom file could use another PR.

Comment (EMsnap) [reply inlong-agent/agent-common/pom.xml]: see https://github.com/apache/inlong/pull/5656

File: .idea/vcs.xml [hunk 3]
```diff
@@ -1,24 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```
Comment (healchow) [inline .idea/vcs.xml:2]: Why remove the license header?

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**